docs/userguide/share_data.rst

```diff
@@ -65,3 +65,34 @@ Data copied from file explorer also appear in the Parsec client. In addition, th
     Parsec client can work while offline (however only data present locally
     are available), synchronization will occur automatically as soon as the
     connection with the server is established.
+
+Assign roles to users in bulk
+-----------------------------
+
+Assigning the same roles to multiple users on multiple workspaces can be a bit annoying. Parsec provides a way to bulk assign the roles of one user to another.
+
+1. In the Users page, right click on the user you want to copy the roles from:
+
+.. image:: screens/bulk_assign_roles_start.png
+    :align: center
+    :alt: Start a bulk assignement
+
+2. Search the user you want to assign the roles to:
+
+.. image:: screens/bulk_assign_roles_select_user.png
+    :align: center
+    :alt: Select the user to assign roles to
+
+3. You're shown a summary of what roles are going to be assigned on each workspace. Click on `Assign roles` to complete the operation.
+
+.. image:: screens/bulk_assign_roles_summary.png
+    :align: center
+    :alt: Bulk assignment summary
+
+.. note::
+
+    1. You can only assign roles on workspaces where you are an Owner or Manager
+    2. Outsider cannot have a role higher than Contributor
+    3. Assigned roles can be downgraded if your own role on the workspace is not high enough (if you're a Manager and you're trying to assign an Owner role, it will be downgraded to Contributor)
```

Comment:
In the spec we've decided it should be downgraded to `Manager`. 
Is there a reason why you've chosen `Contributor`?

Also, I would re-phrase it a little bit:
```suggestion
    3. You cannot assign roles higher than your own. If you have the Manager role in a workspace and you're trying to assign the Owner role, it will be downgraded to Contributor).
```

Reply:
`Managers` can not set other managers, only readers and contributor. Only Owners can assign other Owners.